packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -11,13 +11,26 @@ const DESCRIPTIONS = {
     'If specified, a value of `offline` will cause the connecting client to be granted a refresh token alongside its access token.',
   acrValues:
     'A space-separated list of ACR values specifying acceptable levels of user authentication that the token should have a claim for. Specifying `AAL2` will require the token to have an authentication assuarance level >= 2 which ensures that the user has been authenticated with 2FA before authorizing the requested grant.',
-  active: 'Boolean indicator of weather the presented token is active.',
+  active: 'Boolean indicator of whether the presented token is active.',
+  activePrice:
+    'Whether the price can be used for new purchases. Defaults to true.',
+  amount:
+    'Amount intended to be collected. A positive integer representing how much to charge in the smallest currency unit (e.g. 100 cents to charge $1.00 or 100 to charge ¥100, a zero-decimal currency).',
   assertion: 'A FxA assertion for the signed-in user.',
   authAt:
     'The UTC unix timestamp for the session at which the user last authenticated to FxA server when generating this token, in seconds since the epoch.',
   authPW: 'The PBKDF2/HKDF-stretched password as a hex string.',
+  billingAgreementId: 'A unique identifier for the billing agreement',
+  billingName: 'Full name',
+  brand:
+    'Card brand (e.g. `amex`, `diners`, `discover`, `jcb`, `mastercard`, `unionpay`, `visa`, or `unknown`).',
   bundle:
     'See [**decrypting the bundle**](https://wiki.mozilla.org/Identity/AttachedServices/KeyServerProtocol#Decrypting_the_getToken2_Response) for information on how to extract kA|wrapKb from the bundle.',
+  cancelAtPeriodEnd:
+    'Many subscriptions will automatically renew. If this is the case this value is false, otherwise this value will be true.',
+  cancelledAt: "This is the date the customer's subscription expired.",
+  capabilities:
+    'Comma-separated list of capabilities enabled by this product for all Relying Parties.',
```

Comment:
This is a little misleading -- I only see `capabilities` used in the /profile endpoint, and this is a list of capabilities for the user for all products they are subscribed to, not for a single product.

Also might be helpful to define capability if it isn't already defined elsewhere?